### PR TITLE
Make allowTrailingOptions default to `false`.

### LINKED ISSUE #65: Please bump version to 1.0 when stable
Pre-1.0 version numbers make it look like the package is not yet ready for use.

### PR DESCRIPTION
**Pull request by nex3**
Also release 1.0.0.

Closes #65

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,16 +106,17 @@ print(results['mode']); // debug
 print(results['verbose']); // true
 ```
 
-By default, the `parse()` method stops as soon as it reaches `--` by itself or
-anything that the parser doesn't recognize as an option, flag, or option value.
-If arguments still remain, they go into [ArgResults.rest][rest].
+By default, the `parse()` method additional flags and options to be passed after
```
Comment (munificent) [inline README.md]: "method additional" -> "method allows additional"

File: lib/src/arg_parser.dart [hunk 2]
```diff
@@ -31,16 +31,16 @@ class ArgParser {
...
   /// Creates a new ArgParser.
   ///
-  /// If [allowTrailingOptions] is set, the parser will continue parsing even
-  /// after it finds an argument that is neither an option nor a command.
-  /// This allows options to be specified after regular arguments. Defaults to
-  /// `false`.
-  factory ArgParser({bool allowTrailingOptions: false}) => new ArgParser._(
+  /// If [allowTrailingOptions] is `true` (the default), the parser will parse
+  /// flags and options that appear after positional arguments. If it's `false`,
+  /// the parser will stop parsing as soon as it finds an argument that is
```
Comment (munificent) [inline lib/src/arg_parser.dart]: "will stop" -> "stops".

File: README.md (hunk 1 above)
Comment (nex3) [inline README.md]: Done.

File: lib/src/arg_parser.dart (hunk 2 above)
Comment (nex3) [inline lib/src/arg_parser.dart]: Done.

**Pull request merged**